eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/ImpTest.hs

```diff
@@ -594,13 +596,14 @@ expectMissingGovActionId ::
   GovActionId (EraCrypto era) ->
   ImpTestM era ()
 expectMissingGovActionId govActionId =
+  -- FIXME: @aniketd also check that the id isn't present in enacted or children
```

Comment:
No need. That would be a violation of an invariant which would result in an `assert` failure
```suggestion
```